eclair-core/src/main/scala/fr/acinq/eclair/wire/LightningMessageCodecs.scala

```diff
@@ -135,7 +135,8 @@ object LightningMessageCodecs {
       ("paymentBasepoint" | point) ::
       ("delayedPaymentBasepoint" | point) ::
       ("firstPerCommitmentPoint" | point) ::
-      ("channelFlags" | byte)).as[OpenChannel]
+      ("channelFlags" | byte) ::
+      ("htlcBasepoint" | point)).as[OpenChannel]
```

Comment:
field order has been changed in the specs PR:
   *  [`33`:`payment_basepoint`]
   * [`33`:`delayed_payment_basepoint`]
   * [`33`:`htlc_basepoint`]
   * [`33`:`first_per_commitment_point`]
   * [`1`:`channel_flags`]